library/azure_rm_devtestlabvirtualnetwork.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_devtestlabvirtualnetwork
+version_added: "2.8"
+short_description: Manage Azure DevTest Lab Virtual Network instance.
+description:
+    - Create, update and delete instance of DevTest Lab Virtual Network.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: True
+    lab_name:
+        description:
+            - The name of the lab.
+        required: True
+    name:
+        description:
+            - The name of the virtual network.
+        required: True
+    location:
+        description:
+            - The location of the resource.
+    description:
+        description:
+            - The description of the virtual network.
+    state:
+      description:
+        - Assert the state of the Virtual Network.
+        - Use 'present' to create or update an Virtual Network and 'absent' to delete it.
+      default: present
+      choices:
+        - absent
+        - present
+
+extends_documentation_fragment:
+    - azure
+    - azure_tags
+
+author:
+    - "Zim Kalinowski (@zikalino)"
+
+'''
+
+EXAMPLES = '''
+  - name: Create (or update) Virtual Network
+    azure_rm_devtestlabvirtualnetwork:
+      resource_group: testrg
+      lab_name: mylab
+      name: myvn
+      description: My Lab Virtual Network
+'''
+
+RETURN = '''
+id:
+    description:
+        - The identifier of the resource.
+    returned: always
+    type: str
+    sample: "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourcegroups/testrg/providers/microsoft.devtestlab/
+             mylab/labzimsxxxabcmooo/virtualnetworks/myvn"
```

Comment:
mylab/labzimsxxxabcmooo/? could you change it to 'mytestlab/xx' if 'xx' is necessary?

Reply:
yes, changed to mylab